packages/db-base/src/lib/inMemFileDB.ts

```diff
@@ -67,12 +68,12 @@ interface FileDbSettings {
     backup: BackupOptions;
     change?: ChangeFunction;
     connected: (nameOfServer: string) => void;
-    logger: tools.Logger;
+    logger: Omit<ioBroker.Logger, 'level'>;
```

Comment:
Okay, maybe a type alias `type InternalLogger = Omit<ioBroker.Logger, 'level'>` or similar is a bit more ergonomic to work with.